Clojure-Collections.md

```diff
@@ -0,0 +1,20 @@
+# Clojure - Collections
+One of Clojure's main positives is that it has quite a few different kinds of
+sequence: vectors, lists, hashmaps, *oh my!* And each different type of collection
+has its own upsides, downsides, and a whole host of useful functions that operate
+on them.
+
+To understand the differences between each type of collection, when to use each
+one, and how to convert between them all, it may be necessary to check out
+other types of collection and remind yourself what makes them special.
```

Comment:
I'm pretty sure "type of collection" is correct here, since it's singular. "A dog is a type of animal." "A vector is a type of collection."